features/quizmaster/view_answers.feature

```diff
@@ -14,10 +14,10 @@ Background:
   And I am on the quizmaster page for "Trivia"
 
 Scenario: I correct the answers for a question
-  Given I press the "Send" button for question "1"
+  Given I press the "Send" button for question "What is 2+2?"
   And "Amber Rocks!" has answered question "What is 2+2?" with "4"
   And "Craft Academy" has answered question "What is 2+2?" with "Six"
-  When I click the "Correct" button
+  When I press the correct button for question "What is 2+2?"
```

Comment:
yes because I made a new test where there would be two correct buttons and you want to press the right one